integration/confidential/lib.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+# Copyright (c) 2021, 2022 IBM Corporation
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This provides generic functions to use in the tests.
+#
+set -e
+
+source "${BATS_TEST_DIRNAME}/../../../lib/common.bash"
+FIXTURES_DIR="${BATS_TEST_DIRNAME}/fixtures"
+
+# Toggle between true and false the service_offload configuration of
+# the Kata agent.
+#
+# Parameters:
+#	$1: "on" to activate the service, or "off" to turn it off.
+#
+# Environment variables:
+#	RUNTIME_CONFIG_PATH - path to kata's configuration.toml. If it is not
+#			      export then it will figure out the path via
+#			      `kata-runtime env` and export its value.
+#
+switch_image_service_offload() {
+	# Load the RUNTIME_CONFIG_PATH variable.
+	load_RUNTIME_CONFIG_PATH
+
+	case "$1" in
+		"on")
+			sudo sed -i -e 's/^# *\(service_offload\).*=.*$/\1 = true/g' \
+				"$RUNTIME_CONFIG_PATH"
+			;;
+		"off")
+			sudo sed -i -e 's/^\(service_offload\).*=.*$/#\1 = false/g' \
+				"$RUNTIME_CONFIG_PATH"
+
+			;;
+		*)
+			echo "Unknown option $1"
+			return 1
```

Comment:
Can you use our standard `die()` function here:

```suggestion
			die "Unknown option '$1'"
```